backend/pom.xml

```diff
@@ -136,7 +136,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>${maven.surefire.version}</version>
+                <version>3.0.0-M5</version>
```

Comment:
You can probably ignore the changes in the pom. I was having some issues with IntelliJ so made these changes here.